templates/resources/cloudfront_distribution.md.tmpl

```diff
@@ -0,0 +1,33 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### Cloudfront Distribution with S3 Origin using Origin Access Control
+
+{{ tffile (printf "examples/resources/%s/cloudfront_with_s3_origin_oac.tf" .Name)}}
+
+### Cloudfront Distribution with S3 Origin using Origin Access Identity
+
+{{ tffile (printf "examples/resources/%s/cloudfront_with_s3_origin_oai.tf" .Name)}}
+
+## NOTE: After successful resource creation, edit the Cloufront Origin, origin access from default public setting to use either the OAC or OAI which were created
```

Comment:
please remove this note, we expect the issue to be resolved upstream and hold the merge until then